pkg/controllers/consistency/events.go

```diff
@@ -12,20 +12,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package events
+package consistency
 
 import (
 	v1 "k8s.io/api/core/v1"
 
 	"github.com/aws/karpenter-core/pkg/events"
 )
 
-func InflightCheck(node *v1.Node, message string) []events.Event {
+func CheckEvent(node *v1.Node, message string) []events.Event {
 	return []events.Event{
 		{
 			InvolvedObject: node,
 			Type:           v1.EventTypeWarning,
-			Reason:         "FailedInflightCheck",
+			Reason:         "ConsistencyCheck",
```

Comment:
Should this have the word Failure or some sort of indication that this event is not a good thing? You could argue that the Warning type conveys that, but it seems like it should be conveyed through the name too